extensions/amp-bind/0.1/bind-impl.js

```diff
@@ -715,15 +707,26 @@ export class Bind {
    * @private
    */
   scanAttribute_(attribute, element) {
-    const {tagName} = element;
-    const {name} = attribute;
-    if (name.length > 2 && name[0] === '[' && name[name.length - 1] === ']') {
-      const property = name.substr(1, name.length - 2);
-      if (this.validator_.canBind(tagName, property)) {
+    const tag = element.tagName;
```

Comment:
per https://github.com/ampproject/amphtml/pull/15204#issue-187327075, isn't the old way better? `const {name, tagName } = element`?

Reply:
I wanted to use the shorter `tag` variable for conciseness.